vscode/src/extension.ts

```diff
@@ -65,6 +66,16 @@ export async function activate(context: vscode.ExtensionContext) {
     )
   );
 
+  // signature help
+  context.subscriptions.push(
+    vscode.languages.registerSignatureHelpProvider(
+      "qsharp",
```

Comment:
Could you use `qsharpDocumentFilter` like we do for hover above? (And if you could fix go-to-def as well that would be cherry on top). 

(No practical difference today, but I have plans for this later when I work on notebook cells. Would be nice to have everything consistent now).

Reply:
done